mllib/src/main/scala/org/apache/spark/mllib/clustering/LDA.scala

```diff
@@ -110,14 +125,17 @@ class LDA private (
     this
```

Comment:
As long as you're editing here, can you add a check for docConcentration.length > 0?